src/main/java/de/kekru/struktogrammeditor/view/AuswahlPanel.java

```diff
@@ -24,35 +25,34 @@
 import javax.swing.JLabel;
 import javax.swing.JPanel;
 
-import org.jdom.Document;
+import org.jdom2.Document;
 
 import de.kekru.struktogrammeditor.control.Controlling;
 import de.kekru.struktogrammeditor.control.GlobalSettings;
 import de.kekru.struktogrammeditor.control.Struktogramm;
 import de.kekru.struktogrammeditor.other.Helpers;
 
-
-public class AuswahlPanel extends JPanel implements DropTargetListener, DragGestureListener, DragSourceListener{
+public class AuswahlPanel extends JPanel implements DropTargetListener, DragGestureListener, DragSourceListener {

```

Comment:
Difficult to see the changes here, too with the new formatting

Reply:
I've added a space after DragSourceListener